addons/web_editor/static/src/js/wysiwyg/widgets/image_optimize_dialog.js

```diff
@@ -0,0 +1,223 @@
+odoo.define('wysiwyg.widgets.image_optimize_dialog', function (require) {
+'use strict';
+
+var core = require('web.core');
+var Dialog = require('web.Dialog');
```

Comment:
Not needed here, `ImageOptimizeDialog` is only a class to update existing attachments, it has nothing to do with DOM manipulation.

It could be argued that this file should not be on the `wysiwyg` folde then, but where would be best?

Reply:
Mmh, the we dialog was not designed for DOM manipulation but rather to handle a "save" and "cancel" behavior but since you are overridding the button labels anyway... ok. And not it can stay in that folder.